app/src/test/kotlin/com/omgodse/notally/recyclerview/listmanager/ListManagerMoveTest.kt

```diff
@@ -268,17 +268,34 @@ class ListManagerMoveTest : ListManagerTestBase() {
         setSorting(ListItemSorting.noAutoSort)
         listManager.changeIsChild(1, true, false)
         listManager.changeIsChild(2, true, false)
-        listManager.changeIsChild(4, true, false)
+        listManager.changeIsChild(5, true, false)
         items.printList("Before")
 
-        val newPosition = listManager.simulateDrag(3, 2)
+        dragCallback.simulateDrag(4, 2, "E".itemCount)
         items.printList("After move 3 to 2")
 
-        items.assertOrder("A", "B", "D", "E", "C", "F")
-        "A".assertChildren("B", "D", "E", "C")
+        items.assertOrder("A", "B", "E", "F", "C", "D")
+        "A".assertChildren("B", "E", "F", "C")
         "D".assertChildren()
-        assertEquals(2, newPosition)
-        (changeHistory.lookUp() as ListMoveChange).assert(3, 2, 2, " D(E)")
+        (changeHistory.lookUp() as ListMoveChange).assert(4, 2, 2, " E(F)")
+    }
+
+    @Test
+    fun `endDrag parent with children into other parent with auto-sort enabled`() {
+        setSorting(ListItemSorting.autoSortByChecked)
+        listManager.changeIsChild(1, true, false)
+        listManager.changeIsChild(2, true, false)
+        listManager.changeIsChild(5, true, false)
+        items.printList("Before")
+
+        // TODO: parents cant be moved into other parents' children yet
```

Comment:
Currently there is a bug when auto-sort is enabled, you cant move a parent with its children into another parent's children, I will address this in another PR